app/src/main/kotlin/co/netguru/baby/monitor/client/feature/firebasenotification/FirebaseNotificationSender.kt

```diff
@@ -23,42 +24,44 @@ class FirebaseNotificationSender @Inject constructor(
     private val httpClient: OkHttpClient,
     private val debugModule: DebugModule,
     private val analyticsManager: AnalyticsManager,
-    private val gson: Gson
+    private val gson: Gson,
+    private val feedbackFrequencyUseCase: FeedbackFrequencyUseCase
 ) {
     fun broadcastNotificationToFcm(
-        title: String,
-        text: String,
-        notificationType: NotificationType
+        notificationData: NotificationData
     ): Completable =
         dataRepository.getClientData()
             .doOnSuccess { Timber.i("Client data: $it.") }
             .map { client -> client.firebaseKey }
             .flatMapCompletable { firebaseToken ->
-                analyticsManager.logEvent(Event.ParamEvent.NotificationSent(notificationType))
-                postNotificationToFcm(firebaseToken, title, text, notificationType)
+                analyticsManager.logEvent(Event.ParamEvent.NotificationSent(notificationData.type))
+                postNotificationToFcm(firebaseToken, notificationData)
                     .doOnSuccess { response ->
                         debugModule.sendNotificationEvent(
-                            "Notification posted: Title: $title" +
-                                    " text: $text. Response: ${response.body?.string()}."
+                            "Notification posted: Title: ${notificationData.title}" +
+                                    " text: ${notificationData.text}. Response: ${response.body?.string()}."
                         )
+                        updateFeedbackData(notificationData)
                         Timber.d("Notification posted: $response.")
                     }
                     .ignoreElement()
             }
 
+    private fun updateFeedbackData(notificationData: NotificationData) {
+        feedbackFrequencyUseCase.notificationSent()
+        if (notificationData.feedbackRecordingFile.isNotEmpty())
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/7c069c26782df05dac8a097aa956e351ad95f920/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/firebasenotification/FirebaseNotificationSender.kt#L52"></span>Multi-line if statement was found that does not have braces. These should be added to improve readability.</td>
    </tr>
  </tbody>
</table>